ui/app/src/components/ItemStatesManagement/ItemStatesManagement.tsx

```diff
@@ -268,7 +268,7 @@ export default function ItemStatesManagement(props: ItemStatesManagementProps) {
   return (
     <section ref={rootRef} className={classes.root}>
       <GlobalAppToolbar
-        title={!embedded && <FormattedMessage id="siteTools.itemStates" defaultMessage="Item States" />}
+        title={!embedded && <FormattedMessage id="workflowStates.title" defaultMessage="Workflow States" />}
```

Comment:
Missing removing the duplicate translation (siteTools.itemStates) from json files

Reply:
I did remove the translation, I'll comment on the lines where I removed it